src/YQWizard.h

```diff
@@ -579,6 +580,7 @@ protected slots:
     static std::string		 _releaseNotesButtonId;
     static std::string		 _releaseNotesButtonLabel;
     YQWizardButton *	_helpButton;
+    QAction *           _helpAction;
```

Comment:
That's a matter of convention IMHO. Stuff created with 'new' should be accessible later in case it's needed. But that's purely a matter of style; the parent QObject will delete it in its destructor one way or the other.

As for ABI, those widgets in the concrete UIs were never meant to be inherited from (even less something as specialized as this YQWizard); that's why they are not PIMPL'ed, either.